scripts/init/opx-front-panel-ports.service

```diff
@@ -1,13 +1,14 @@
 [Unit]
 Description=Front panel port object access handler
 After=opx-cps.service opx-nas.service
-Wants=opx-cps.service opx-nas.service
+Requires=opx-cps.service opx-nas.service
 OnFailure=service_onfailure@%n.service
 
 [Service]
 Type=notify
 EnvironmentFile=/etc/opx/opx-environment
 ExecStart=/usr/bin/python -u /usr/bin/base_nas_front_panel_ports.py
+TimeoutStartSec=90
```

Comment:
May I ask the reasoning behind it? You want to check if this service starts up in 90 sec, if not restart it?

Is this number standard?